cpp/dolfinx/mesh/utils.h

```diff
@@ -108,5 +108,24 @@ Eigen::Array<std::int32_t, Eigen::Dynamic, 1> locate_entities_boundary(
         const Eigen::Ref<const Eigen::Array<double, 3, Eigen::Dynamic,
                                             Eigen::RowMajor>>&)>& marker);
 
+/// Compute the geometry indices of given entities from the mesh geometry
```

Comment:
The doc description isn't clear. Does it get geometry indices for the vertices of the mesh entities?